scripts/export_comments.py

```diff
@@ -90,6 +91,11 @@ def get_comment_text(submission):
 
     try:
         submissions = get_all_submissions(survey_id, period)
+        if survey_id == '181':
+            submissions.extend(get_all_submissions('182', period))
+            submissions.extend(get_all_submissions('183', period))
+            submissions.extend(get_all_submissions('184', period))
+            submissions.extend(get_all_submissions('185', period))
```

Comment:
I think the specific stuff should live in the `get_all_submissions` function instead of the main block of code.  It makes it easier to understand the broad strokes of what the script is doing.

The code should also be done slightly differently too.  Instead of doing a thing 4 times, do it in a for loop to make it tidier. So the code in the get_all_submissions` function will look something like 
```python
if survey_id == '181':
    for survey_id in ['182', '183', '184', '185']:
        // Get the records
    return records

survey_records = session.query(SurveyResponse).filter(SurveyResponse.data['survey_id'].astext == survey_id)
records = survey_records.filter(SurveyResponse.data['collection']['period'].astext == period).all()
print(f"Retrieved {len(records)} submissions for {survey_id}")
return records
```